person.rb

```diff
@@ -1,8 +1,12 @@
 # frozen_string_literal: true
 
+require './nameable'
+require './decorator'
```

Comment:
- [ ] OPTIONAL
It's recommended to use `require_relative` for classes within a structure and rather use `require` for _gems_ and other packer imports. Kindly endeavor to improve upon this aspect